### PR TITLE
[IMP] m2o_to_m2m: Join SQL queries in one

### PR DESCRIPTION
**Pull request by pedrobaeza**
Improvement according https://github.com/OpenUpgrade/OpenUpgrade/pull/234/files#r26291892

### CODE REVIEW

**Comment by hbrunn** [comment]
:+1:

**Comment by StefanRijnhart** [comment]
Also here it makes sense to combine the two queries: https://github.com/pedrobaeza/OpenUpgrade/blob/7.0-m2o_to_m2m-imp/openerp/openupgrade/openupgrade.py#L441-L453

**Comment by pedrobaeza** [comment]
@StefanRijnhart, I haved joined both queries, please review it.

**Comment by pedrobaeza** [comment]
The 8.0 version is in #261

**Comment by StefanRijnhart** [comment]
Thanks, :+1:

File: openerp/openupgrade/openupgrade.py [hunk 1]
```diff
@@ -438,19 +438,14 @@ def m2o_to_m2m(cr, model, table, field, source_field):
...
-                   'field': source_field,
-                   })
     rel, id1, id2 = osv.fields.many2many._sql_names(model._columns[field],
                                                     model)
-    for row in cr.fetchall():
-        cr.execute(
-            "INSERT INTO %s (%s, %s) "
-            "VALUES (%s, %s)" %
-            (rel, id1, id2, row[0], row[1]))
+    cr.execute(
```
Comment (bwrsandman) [inline openerp/openupgrade/openupgrade.py]: This might be a good place to use `logged_query`. What do you think?

**Comment by pedrobaeza** [comment]
@bwrsandman, done!

**Comment by bwrsandman** [comment]
:+1: Thanks!

**Pull request merged**